### PR TITLE
add cellular_raza to showcase exampleSite

### PR DESCRIPTION
**Pull request by jonaspleyer**
Hi,
Thank you for this really nice hugo theme. I would love to add the documentation website of my simulation tool `cellular_raza` to the showcase site.
I was unsure where to put the image so I simply included it in the static folder but I assume that this is not the correct place. Please let me know where to put it.

### CODE REVIEW

File: exampleSite/static/images/cellular_raza.png
Comment (imfing) [inline exampleSite/static/images/cellular_raza.png:1]: please remove this

File: exampleSite/content/showcase/index.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ Open source projects powered by Hextra
 </p>
 
 {{< cards >}}
+  {{< card
+        link="https://github.com/jonaspleyer/cellular_raza"
+        title="cellular_raza"
+        image="/images/cellular_raza.png"
```
Comment (imfing) [inline exampleSite/content/showcase/index.md:18]:
```suggestion
        image="https://github.com/user-attachments/assets/f24c6455-b70a-419b-b025-e3d60101b673"
```

**Review by imfing — CHANGES_REQUESTED** [review]
@jonaspleyer thank you, the project site looks fantastic! 

I'd prefer uploading images to GitHub PR or Issues comment, and use the link generated by GitHub (I did that already, please check the comment below)

File: exampleSite/static/images/cellular_raza.png
Comment (jonaspleyer) [reply exampleSite/static/images/cellular_raza.png:1]: Thanks for the changes. I have commited them.

**Pull request merged**